examples/http-client-header-parameter/http_client_header_parameter_header.bal

```diff
@@ -0,0 +1,13 @@
+Album payload = {
+    title: "Blue Train",
+    artist: "John Coltrane"
+};
+
+map<string> headers = {
+    "Accept": mime:APPLICATION_JSON
+};
+
+// Sends a `POST` request to the "/albumId" resource.
+// The query parameter can be provided as parameters in the `post` method invocation.
+string albumId = check albumClient->/addAlbum.post(payload, headers);
```

Comment:
In this case also lets give the headers in line

Reply:
Updated